### PR TITLE
JNG-4547 Release epsilon-maven-plugin

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4547" title="JNG-4547" target="_blank">Sub-taskJNG-4547</a>  Release epsilon-maven-plugin
  </td></table>
  <br />

### CODE REVIEW

**Review by bencegelei — CHANGES_REQUESTED** [review]
java 11?

File: .buildkite/build.sh [hunk 1]
```diff
@@ -4,5 +4,5 @@ set -ex
 MAVEN_PROJECT_VERSION=$(python -c "import xml.etree.ElementTree as ET; print(ET.parse(open('pom.xml')).getroot().find('{http://maven.apache.org/POM/4.0.0}version').text)")
 PROJECT_VERSION=$(echo $MAVEN_PROJECT_VERSION | sed "s/-SNAPSHOT/-$BUILDKITE_BUILD_NUMBER/")
 mvn versions:set -DnewVersion=$PROJECT_VERSION
-mvn clean deploy -DaltDeploymentRepository=blackbelt-nexus-distribution::default::${NEXUS3_URL}/repository/maven-epsilon-runtime-release
+mvn clean deploy -DaltDeploymentRepository=blackbelt-nexus-distribution::default::${NEXUS3_URL}/repository/maven-epsilon-maven-plugin-release
```
Comment (noherczeg) [inline .buildkite/build.sh]: "epsilon-maven-plugin"

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: .buildkite/build.sh (hunk 1 above)
Comment (gaborflorian) [reply .buildkite/build.sh]: [34b0901](https://github.com/BlackBeltTechnology/epsilon-maven-plugin/pull/42/commits/34b090102772661dce26a6a5248d80818360b7e0)

File: .buildkite/build.sh [hunk 2]
```diff
@@ -4,5 +4,5 @@ set -ex
 MAVEN_PROJECT_VERSION=$(python -c "import xml.etree.ElementTree as ET; print(ET.parse(open('pom.xml')).getroot().find('{http://maven.apache.org/POM/4.0.0}version').text)")
 PROJECT_VERSION=$(echo $MAVEN_PROJECT_VERSION | sed "s/-SNAPSHOT/-$BUILDKITE_BUILD_NUMBER/")
 mvn versions:set -DnewVersion=$PROJECT_VERSION
-mvn clean deploy -DaltDeploymentRepository=blackbelt-nexus-distribution::default::${NEXUS3_URL}/repository/maven-epsilon-runtime-release
+mvn clean deploy -DaltDeploymentRepository=blackbelt-nexus-distribution::default::${NEXUS3_URL}/repository/epsilon-maven-plugin-release
```
Comment (noherczeg) [inline .buildkite/build.sh]: We don't use buildkite any more. File should be deleted.

File: pom.xml [hunk 3]
```diff
@@ -544,5 +701,151 @@
...
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>first</id>
+                                <goals>
+                                    <goal>update-file-header</goal>
+                                </goals>
+                                <phase>process-sources</phase>
+                                <configuration>
+                                    <licenseName>apache_v2</licenseName>
```
Comment (noherczeg) [inline pom.xml]: Should be epl.

**Review by noherczeg — CHANGES_REQUESTED** [review]
Wrong license file

**Review by bencegelei — APPROVED** [review]

**Review by robertcsakany — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]

**Pull request merged**